### PR TITLE
Use experimental instead of prototype

### PR DESCRIPTION
**Pull request by malomarrec**
Clarify that we use Experimental and Beta to highlight feature maturity. Prototype is replaced by Experimenal.

### CODE REVIEW

**Review by quinnkeast — APPROVED** [review]

**Comment by dan-mckean** [comment]
Are we using "experimental" to more actively dissuade people?

Prototype sounds somewhat less risky than experimental.

**Comment by malomarrec** [comment]
@dan-mckean see this [thread](https://sourcegraph.slack.com/archives/C0C324C91/p1626245314372400), which has some answers and also some more questions :)

**Review by dan-mckean — APPROVED** [review]

**Comment by malomarrec** [comment]
Also see: https://github.com/sourcegraph/about/pull/4528

File: doc/code_insights/language_insight_quickstart.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Enter repositories in the repository URL format, like `github.com/Sourcegraph/So
 
 The form field will validate that you've entered the repository correctly. 
 
-In the current prototype version, you can only set up a language insight for one repository at a time. (Want us to prioritize the ability to analyze multiple repositories in one pie chart? [Please do let us know](mailto:feedback@sourcegraph.com).)
+In the current experimental version, you can only set up a language insight for one repository at a time. (Want us to prioritize the ability to analyze multiple repositories in one pie chart? [Please do let us know](mailto:feedback@sourcegraph.com).)
```
Comment (Joelkw) [inline doc/code_insights/language_insight_quickstart.md:31]:
```suggestion
In the current Beta version, you can only set up a language insight for one repository at a time. (Want us to prioritize the ability to analyze multiple repositories in one pie chart? [Please do let us know](mailto:feedback@sourcegraph.com).)
```

(This should have been beta anyway, thanks for the catch!)

**Review by Joelkw — APPROVED** [review]
Thanks for updating!

**Pull request merged**